core_backend/app/contents/models.py

```diff
@@ -116,6 +113,53 @@ def __repr__(self) -> str:
         )
 
 
+class ContentForQueryDB(Base):
```

Comment:
Not sure about the location of this + its class name + table name, doesn't seem clear to me... I'm wondering if this should actually be defined beside QueryResponseDB inside the question_answer module and be called QueryResponseContentDB. This would logically make sense (these records are added only after a search is done) and the name would be clear too.


If it HAS to stay here, about  ContentReturnedDB or ContentSearchHistoryDB or ContentSearchResultDB or smt like that? With the corresponding SQL table name. I'd vote for moving it though.

Reply:
Make sense. I'll move it